ibm_catalog.json

```diff
@@ -482,33 +504,49 @@
               }
             },
             {
-              "key": "key_protect_instance_cbr_rules"
+              "key": "key_protect_instance_cbr_rules",
+              "type": "array",
+              "custom_config": {
+                "type": "textarea",
+                "grouping": "deployment",
+                "original_grouping": "deployment"
+              }
+            },
+            {
+              "key": "provider_visibility",
+              "options": [
+                {
+                  "displayname": "private",
+                  "value": "private"
+                },
+                {
+                  "displayname": "public",
+                  "value": "public"
+                },
+                {
+                  "displayname": "public-and-private",
+                  "value": "public-and-private"
+                }
+              ],
+              "hidden": true
             }
           ],
           "architecture": {
-            "description": "This architecture supports creating and configuring a Key Protect instance.",
+            "description": "This architecture supports creating and configuring a Key Protect instance, key rings and keys.",
             "features": [
               {
-                "title": "Key Protect instance",
-                "description": "Creates and configures a Key Protect instance."
-              },
-              {
-                "title": "Key Rings and Keys",
-                "description": "Creates Key Rings and Keys for either Key Protect or Hyper Protect Crypto Services instances."
-              },
-              {
-                "title": "Context-Based Restriction rules",
-                "description": "Creates Context-Based Restriction rules for Key Protect instance."
+                "title": "",
+                "description": "Configured to use IBM secure by default standards that can't be changed."
               }
             ],
             "diagrams": [
               {
                 "diagram": {
                   "caption": "IBM Key Protect architecture",
-                  "url": "https://raw.githubusercontent.com/terraform-ibm-modules/terraform-ibm-kms-all-inclusive/main/reference-architecture/key_protect.svg",
+                  "url": "https://raw.githubusercontent.com/terraform-ibm-modules/terraform-ibm-kms-all-inclusive/main/reference-architecture/deployable-architecture-key_protect.svg",
```

Comment:
Please rename the diagram correctly - either use hyphens across(preferred) or use underscore only.
`deployable-architecture-key_protect.svg"` -> `deployable-architecture-key-protect.svg"`